format_ideas/gaussian1d-option-1.yaml

```diff
@@ -0,0 +1,70 @@
+# Option 1: Use the same representation for minimal models and models with units/constraints
+
+
+!transform/tags/gaussian1d-1.0
+  name: Minimal single model
+
+  parameters:
+    amplitude: !transform/tags/parameter-1.0
+      value: 1.0
+
+    mean: !transform/tags/parameter-1.0
+      value: 2.0
+
+    stddev: !transform/tags/parameter-1.0
+      value: 3.0
+
+  inputs: !!omap
+    x: !transform/tags/input-1.0 {}
+
+  outputs: !!omap
+    y: !transform/tags/output-1.0 {}
+
+
+!transform/tags/gaussian1d-1.0
+  name: Single model with units and bounds and constraints
+
+  parameters:
+    amplitude: !transform/tags/parameter-1.0
+      value: 1.0
+      unit: !core/tags/unit-1.0 m
+      bounds: [0.0, 2.0]
+
+    mean: !transform/tags/parameter-1.0
+      value: 2.0
+      unit: !core/tags/unit-1.0 s
+      fixed: true
+
+    stddev: !transform/tags/parameter-1.0
+      value: 3.0
+      unit: !core/tags/unit-1.0 s
+      bounds: [2.0, 4.0]
+
+  inputs: !!omap
+    x: !transform/tags/input-1.0
```

Comment:
The problem with this approach is that one needs something that indicates the order of inputs and output. No examples show more than one of these. An unless order is implied by the order in yaml (which it could be I suppose), this wouldn't work since it isn't explicit in the attribute name. What was your thinking on this case?

Reply:
The `!!omap` tag is YAML's way of designating a mapping as ordered.  So here the order is supposed to be maintained by the reader.  But I'm leaning towards the list anyway, since the `!!omap` feature is a bit obscure.